tests/client/core/components/TestAddonCard.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { renderIntoDocument } from 'react-addons-test-utils';
+
+import AddonCard from 'core/components/AddonCard';
+
+
+// eslint-disable-next-line react/prop-types
+const FakeAddonComponent = ({ addon }) => (<div>{addon.name}</div>);
```

Comment:
This is rendered in a `<ul>`, maybe it should be an `<li>` to read more clearly?